### DISCUSSION TITLE
Any success with PTZ on reolink TrackMix PoE?

### DISCUSSION THREAD

**Discussion by grainsoflight** [blakeblackshear/frigate#9739] [General]
I am able to control the PTZ on the TrackMix PoE though autotracking becomes disabled and reports that FOV Relative Movement is unsupported. Am I doing something wrong or is this just a cam that will not work with this feature?

**Comment by NickM-27** [comment]
Reolink cameras do not support the necessary features

**Comment by grainsoflight** [comment]
Darn. OK thanks

**Reply by Poole-CA** [reply]
I don't own a PTZ camera, but I've looked a bit at the Reolink Trackmix. Why do you need Frigate to control PTZ when the camera already has what seems like pretty good tracking built in?

Other than tracking, how do you like the camera? Are you able to get usable  streams into Frigate?

**Reply by grainsoflight** [reply]
I guess having all of those controls in one place is the main draw, plus presumably frigate wouldn't get confused by the camera moving without it's knowledge. The trackmix works fine when viewing MSE streams. I just recently got it working with WebRTC through go2RTC's ffmpeg hardware encoding.

**Reply by eon0613** [reply]
hi,

would you be willing to share your config, im trying to make my TRACKMIX POE  PTZ work, i dont care about the AUTOTRACKING
Just want to b able to move it using manual control

thanks

**Reply by gio-dot** [reply]
Hi, can you share your config?

Plus i have some question:

1. Can you manage both cameras inside the cam (wide and narrow lens) in frigate?
2. Does each camera inside the cam have high and low resolution flows?

**Reply by grainsoflight** [reply]
@eon0613 @gio-dot, I just configure the ONVIF credentials for the camera. 1. Each camera needs to be configured as a seperate camera, 2. yes there are two streams for each camera

 ```
   onvif:
      host: 192.168.5.2
      user: user
      password: password
```

**Reply by eon0613** [reply]
thanks for your reply

i was able to add the ONVIF in my config and running fine, but i dont see the control for the PTZ?

**Reply by eon0613** [reply]
@gio-dot 

here's my config

go2rtc:
  streams:
    front_yard:
    - rtsp://admin:Leianne@1994@192.168.1.99:554/h264Preview_02_sub
    formal_lvgrm:
    - rtsp://admin:Leianne@1994@192.168.1.99:554/h264Preview_03_sub
    front_doorbell:
    - rtsp://admin:Leianne@1994@192.168.1.99:554/h264Preview_05_sub
    garage:
    - rtsp://admin:Leianne@1994@192.168.1.99:554/h264Preview_01_sub
    garage_door:
    - rtsp://admin:Leianne@1994@192.168.1.99:554/h264Preview_04_sub
    front_door_side:
    - rtsp://admin:Leianne@1994@192.168.1.99:554/h264Preview_06_sub
    webrtc:
    candidates:
    - 192.168.1.91:8555
    - stun:8555

timestamp_style:
  position: tl
  format: '%m/%d/%Y %H:%M:%S'
  color:
    red: 255
    green: 255
    blue: 255
  thickness: 2
  effect: shadow
# ffmpeg:
  # hwaccel_args: preset-nvidia-h264
cameras:
  front_yard:
    ffmpeg:
      inputs:
      - path: rtsp://127.0.0.1:8554/front_yard
        input_args: preset-rtsp-restream
        #hwaccel_args: preset-nvidia-h265
        roles:
        - detect
    onvif:
      host: rtsp://127.0.0.1:8554/front_yard
      port: 8000
      user: admin
      password: Leianne@1994    
    detect:
      width: 896
      height: 512
      fps: 10
    objects:
      track:
      - person

      filters:
        person:
          mask:
          - 173,315,234,306,240,272,231,244,173,257 
    snapshots:
      enabled: true
      timestamp: true
      bounding_box: true
 #     required_zones:
  #      - front_yard
      crop: true
      height: 500
      retain:
        default: 3
#    zones:
 #       front_yard_zone:
  #      coordinates: 505,131,604,128,611,459,466,459,361,480,188,480,298,284,179,261,242,210,320,138
        #inertia: 1

    motion:
      mask:
      - 640,0,640,41,265,40,262,0
      - 180,308,240,294,246,247,174,261

**Reply by grainsoflight** [reply]
Is ONVIF enabled on your camera?

**Reply by grainsoflight** [reply]
Your ONVIF host should be an IP address, not an RTSP stream

**Reply by grainsoflight** [reply]
@eon0613  Is there a reason all of your cameras have the same IP address?

**Reply by ragaimeena** [reply]
good morning. I just bought this camera. would you kindly consider sharing your config in a format that can be pasted into the container config ( I am having a hard time with the indentations)
Also are you able to control PTZ and all from the frigate interface?
Thank you

**Reply by JoshuaSeidel** [reply]
Are you attaching the cameras to frigate through a NVR?

**Reply by ragaimeena** [reply]
No
